rollup.config.js

```diff
@@ -2,7 +2,9 @@ import cleanup from 'rollup-plugin-cleanup';
 import json from '@rollup/plugin-json';
 import resolve from '@rollup/plugin-node-resolve';
 import {terser} from 'rollup-plugin-terser';
-import {version, homepage} from './package.json';
+import { readFileSync } from "fs";
+
+const {version, homepage} = JSON.parse(readFileSync('./package.json'));
```

Comment:
Can be changed to:
```js
import * as packageJson from './package.json' assert {type: "json"};
```

But then the rollup file needs to be excluded from eslint, not sure what the better/nicer solution is

Reply:
I think this way is just fine. Import assertions aren't standardized yet and are still at the proposal phase